packages/devtools_app/lib/src/network/network_screen.dart

```diff
@@ -33,11 +32,7 @@ class NetworkScreen extends Screen {
   static const recordingInstructionsKey = Key('Recording Instructions');
 
   @override
-  Widget build(BuildContext context) {
-    return !serviceManager.connectedApp.isDartWebAppNow
-        ? const NetworkScreenBody()
-        : const DisabledForWebAppMessage();
-  }
+  Widget build(BuildContext context) => const NetworkScreenBody();
```

Comment:
shouldn't we still be disabling this for dart web apps? is `requiresDartVm: true` now the equivalent of the `!serviceManager.connectedApp.isDartWebAppNow` condition?

Reply:
Yep, I somehow missed the condition on this. Fixed!

> is `requiresDartVm: true` now the equivalent of the `!serviceManager.connectedApp.isDartWebAppNow` condition?

Yep, they're the same. We could invert it, but the naming might be worse (`requiresNonWebApp`, `disabledForWebApp`?). I can update if there are any preferences!